docs/docs/dev/sidebar.json

```diff
@@ -143,6 +143,7 @@
     "children": [
       "documentation/http-api",
       "documentation/kubernetes-annotations",
+      "documentation/dpp-specification",
```

Comment:
Can we put this in the `reference` folder? Trying to stop putting everything in the same `documentation` directory